### ISSUE TITLE
A11y_api-catalog_ System.Activities.Debugger.Symbol _ Screen Reader: Narrator/NVDA is not reading name for “Declarations” table

### ISSUE THREAD

**Issue by SaisreeDandamudi** [terrajobst/apisof.net#14]
[Check out Accessibility Insights] - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:** 
#A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_api-catalog; #MAS1.3.1; #Screen Reader; #C+E-H; #api-catalog; #Narrator; #NVDA; #Edge; #Chrome; #Benchmark; #A11ySev2;

**Environment Details:** 
URL: https://apisof.net/

**Browser:**
Microsoft Edge 44.18362.267.0
Chrome Version 76.0.3809.100

**Screen Readers:**
Narrator
NVDA

**Re-pro Steps:**
1.	Start Narrator/NVDA
2.	Hit the [URL](https://apisof.net/)
3.	Tab till “Browse” button and press enter
4.	Select any of the Catalog present in the list
5.	Turn on the scan mode and press “T” key (short cut to navigate to tables) and check whether screen reader is reading the name for the table

**Actual Result:** 
In scan mode when user uses the short cut “T” to navigate to tables in the page, Narrator/NVDA is not reading name for “Declarations” table

**Expected result:** 
Narrator/NVDA should read the table name when user uses the short cut “T” in scan mode

**User Impact:** 
Screen Reader users will not get to know about the table if screen reader doesn’t read the name for the table

![NVDA is not reading the name for table](https://user-images.githubusercontent.com/51435580/63574579-b2b51b80-c5a5-11e9-8cf9-bcf397c72a0b.png)
[10.Narrator is not reading any information about graph.pptx](https://github.com/terrajobst/apisof.net/files/3533821/10.Narrator.is.not.reading.any.information.about.graph.pptx)

**Comment by terrajobst** [comment]
The web site no longer exists.
